src/parser/resolve.js

```diff
@@ -30,31 +31,55 @@ export const resolveDocuments = (
     return {resolved, errors};
 };
 
+export const getPathWithExtension = (pathWithoutExtension: string): string => {
```

Comment:
Is it not possible to share this function?

Reply:
I noticed it right after putting up the PR.  This is fixed now.